src/redis.go

```diff
@@ -82,6 +86,15 @@ func main() {
 	fatalIfErr(i.Publish())
 }
 
+func entity(i *integration.Integration, args *argumentList) (*integration.Entity, error) {
+	if args.RemoteMonitoring {
+		n := fmt.Sprintf("%s:%d", args.Hostname, args.Port)
+		return i.Entity(n, entityRemoteType)
```

Comment:
Not sure about this namespace. I was between `redis` or `remote`. What do you understand for `namespace` or `entity type` in entities?

Reply:
I think namespace should be `redis`. Right @ikavanaghNR ?